Jellyfin.Plugin.SmartPlaylist/Configuration/config.js

```
@@ -23,6 +23,274 @@
+        var days = [];
+        for (var i = 1; i <= 31; i++) {
+            var suffix = '';
+            if (i === 1 || i === 21 || i === 31) suffix = 'st';
+            else if (i === 2 || i === 22) suffix = 'nd';
+            else if (i === 3 || i === 23) suffix = 'rd';
+            else suffix = 'th';
+            
+            days.push({ value: i.toString(), label: i + suffix });
+        }
+        var html = '';
+        for (var i = 0; i < days.length; i++) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)